portfolio/src/main/webapp/script.js

```diff
@@ -63,4 +65,36 @@ function loadTasks() {
       taskListElement.appendChild(createTaskElement(task));
     })
   });
-}
\ No newline at end of file
+}
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+}
+
+// function createTaskElement(task) {
+//   const taskElement = document.createElement('li');
+//   taskElement.className = 'task';
+
```

Comment:
Delete?

Reply:
not sure yet, I think I'll need this code to delete my comments but I haven't decided yet because my comments aren't displaying properly